LiveSplit.AutoSplitters.xml

```diff
@@ -24057,4 +24057,14 @@
         <ScriptType>AutoSplittingRuntime</ScriptType>
         <Description>Autosplitter available (by SunglassesEmoji)</Description>
     </AutoSplitter>
+    <AutoSplitter>
+        <Games>
+            <Game>Brotato</Game>
+        </Games>
+        <URLs>
+            <URL>https://github.com/RobTheFiveNine/brotato-autosplitter/releases/download/v1.0.0/brotato.asl</URL>
```

Comment:
It's not recommended to provide the link like this. You will need to PR this repo every time you update the autosplitter.

Reply:
No problem - I'll push a change in a moment to make it directly reference the main branch

Reply:
I've changed it to point to https://raw.githubusercontent.com/RobTheFiveNine/brotato-autosplitter/refs/heads/main/brotato.asl 👍🏼